java/src/main/java/com/microsoft/onnxruntime/MapInfo.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
> Copyright © 2019, Oracle and/or its affiliates. All rights reserved. [](start = 2, length = 69)

We will need to clarify licensing.

Reply:
I'm happy to add the MIT license notice underneath the copyright, but I have to put the copyright statement on as part of our internal open source process.

Reply:
Okay. It is fine for Oracle Copyright(c) remain. We need to  1) add MIT license 2) change package domains to ai.onnxruntime

---
In reply to: [340884932](https://github.com/microsoft/onnxruntime/pull/2215#discussion_r340884932) [](ancestors = 340884932)

Reply:
Done.